server/controllers/konnectors.coffee

```diff
@@ -77,6 +50,24 @@ module.exports =
 
             res.status(204).send konnector
 
+    update: (req, res, next) ->
+        # Don't update connector if an import is already running.
+        if req.konnector.isImporting
+            res.send 400, message: 'konnector is importing'
```

Comment:
I agree, a 409 woold be more accurate, or even a 423 : Locked ; or 429 : Too Many Requests

To be honnest, I just copy past this from the _import_ method (https://github.com/jacquarg/konnectors/blob/61d38892dd43284bca15ecef349fb281d44c07d7/server/controllers/konnectors.coffee#L78-L81 ). We should the both response, but I am not sure that it will not break something in the client.